### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by VitaliiLeshchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return ((result1 > result2 & bet1 > bet2)
+                || (result1 < result2 & bet1 < bet2)
+                || (result1 == result2 & bet1 == bet2))
+                ? ((result1 == bet1 & result2 == bet2)
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: Great job! But let's first check this condition and after that the others

**Review by dmytro-hryhoruk — COMMENTED** [review]
Great job!

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 1 above)
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/FootballMatch.java]: done. but i don't see difference

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1 && result2 == bet2) ? 2 :
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: '?' should be on a new line

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 3]
```diff
@@ -15,6 +15,9 @@
...
+                (result1 > result2 & bet1 > bet2)
+                || (result1 < result2 & bet1 < bet2)
+                || (result1 == result2 & bet1 == bet2) ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: same

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 2 above)
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**